src/base/check.h

```diff
@@ -54,14 +54,7 @@
 #define CVC5_PREDICT_TRUE(x) x
 #endif
 
-#ifdef __has_cpp_attribute
-#if __has_cpp_attribute(fallthrough)
 #define CVC5_FALLTHROUGH [[fallthrough]]
```

Comment:
Why is this in `check.h`?

Reply:
Yeah, I was wondering about that, too. Moved.